README.md

```diff
@@ -20,6 +20,13 @@ Welcome to the Swarm.... Bzzz Bzzzz Bzzzz
    |                '  ' '
 ```
 
+## DISCLAIMER
```

Comment:
there is already a disclaimer in the `LICENSE` file. I don't think this is necessary as the license file should already cover this legally. I would also advise using the disclaimer in the license file if you really want to print it in the terminal. It is not needed to be explicit on what can happen as a result of using `bee`.